app/routes/circulars._archive._index.tsx

```diff
@@ -291,7 +324,160 @@ export default function () {
           </Button>
         </Form>
         {featureCircularsFilterByDate && (
-          <DateSelectorButton startDate={startDate} endDate={endDate} />
+          <>
+            <DateSelectorButton
+              startDate={startDate}
+              endDate={endDate}
+              onClick={() => setShowContent(!showContent)}
+              expanded={showContent}
+            />
+            {showContent && (
+              <DetailsDropdownContent className="margin-top-1">
+                <div className="usa-radio maxw-card-xlg">
+                  <div className="display-flex flex-row">
+                    <div className="display-flex flex-column flex-align-start margin-1">
+                      <Radio
+                        id="radio-alltime"
+                        name="radio-date"
+                        value="undefined"
+                        defaultChecked={true}
+                        label="All Time"
+                        className="usa-search__filter-button"
+                        onChange={(e) => {
+                          setFuzzyTime(e.target.value)
+                        }}
+                      ></Radio>
```

Comment:
Use self-closing element syntax:

```suggestion
                     / >
```

and the same for the other radio buttons.

Reply:
implemented in [8a26a4e](https://github.com/nasa-gcn/gcn.nasa.gov/pull/1482/commits/8a26a4ee4ab4d1e98fc5a50458438633f7ddf4fb)